cpp/ql/src/Security/CWE/CWE-020/IRUntrustedDataToExternalAPI.qhelp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Using unsanitized untrusted data in an external API can cause a variety of security issues. This query reports 
+external APIs that use untrusted data. The results are not filtered so that you can audit all examples. The query provides data for security reviews of the application and you can also use it to identify external APIs that should be modeled as either taint steps, or sinks for specific problems.</p>
```

Comment:
> The results are not filtered so that you can audit all examples.

What does this sentence mean? Should I parse it as "The results are not (filtered so that you can audit all examples)" or as "The results are not filtered (so that you can audit all examples)"? I'm not even sure what filtering a reader is supposed to associate with this query.

Reply:
My intention was the latter parse tree, but I can see that it's ambiguous. Would it disambiguate the sentence to write `The results are not filtered. This makes it possible to audit all examples.`? If we're diverging from the qhelp in Java I'm also completely okay with removing that sentence altogether.

Reply:
(Potentially) fixed in 8e5e2f864f33a202e47e42bbd3cf41f5aeb3fab8

Reply:
A less clunky rewrite would be "The results are not filtered, so you can audit all examples." The comma makes the parsing unambiguous.

Reply:
That's indeed much better! Done in f16591dffc4a538a32700c077487fcae8fcf26ca.